storage/simulation/script.go

```diff
@@ -282,6 +301,15 @@ func (s *Script) parse(scriptFile string) error {
 			}
 		}
 
+		// Get the percentage value.
+		if len(elements) > 5 {
+			if details.percent, err = strconv.Atoi(elements[5]); err != nil {
+				return util.Errorf("line %d PERCENT could not be parsed: %s", lineNumber, elements[5])
+			}
```

Comment:
Removed the clamping and added error checking.
And I'm going to leave it as an int for now, doesn't seem like a need to make the change.